demos/python_demos/clip/requirements.txt

```diff
@@ -0,0 +1,4 @@
+tritonclient[grpc]==2.37.0.9383150
+openvino
+transformers
```

Comment:
why openvino, transformers and pillow. there are not used on the client

Reply:
We need it to download model and covert it.